assets/js/theme/global.js

```diff
@@ -24,11 +24,11 @@ export default class Global extends PageManager {
         window.lazySizesConfig = window.lazySizesConfig || {};
         window.lazySizesConfig.loadMode = 1;
 
+        cartPreview();
```

Comment:
If we can load the cart qty from "cache" I want to do that ASAP, so making this first.

The network activity this kicks off is async.